maquetado/css/styles.css

```diff
@@ -34,3 +33,35 @@
     display: none;
   }
 }
+
+.features {
+  display: flex;
+  margin-top: 100px;
+  flex-flow: row wrap;
+  justify-content: space-around;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+b {
```

Comment:
don't target html elements, use classes instead.

you can change this to:
```
* {
  font-family: 'Open Sans', sans-serif;
}
```

so all the elements have the right font